### PR TITLE
Lgtm alert x 1

### PR DESCRIPTION
**Pull request by macfarla**
Fix 1 LGTM alerts 
* check null privacyPluginParams in BesuCommand

## Changelog

- [x] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

**Review by fab-10 — CHANGES_REQUESTED** [review]
I have proposed some changes to make clear that Jackson closes the stream, and also improve the closing behavior of `JsonResposeStream` in another PR https://github.com/hyperledger/besu/pull/3421

File: ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpService.java [hunk 1]
```diff
@@ -626,9 +626,10 @@ private void handleJsonSingleRequest(
               response.end(EMPTY_RESPONSE);
             } else {
               try {
-                JSON_OBJECT_WRITER.writeValue(
-                    new JsonResponseStreamer(response, routingContext.request().remoteAddress()),
-                    jsonRpcResponse);
+                final JsonResponseStreamer jsonResponseStreamer =
```
[the remark this answers is not in the document]
Comment (macfarla) [reply ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpService.java]: 👍

[the remark this answers is not in the document]
Comment (macfarla) [reply ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpService.java]: so LGTM is just plain wrong about this?

**Comment by macfarla** [comment]
idk why the main besu lgtm isn't working but if I look at "my alerts" it shows this one - is this rule incorrect?
https://lgtm.com/rules/2980072/
Screen Shot 2022-02-15 at 3 11 54 pm

File: ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpService.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (fab-10) [reply ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpService.java]: Idk how LGTM works, but it seems that it does not understand that the output stream is managed by Jackson.

**Review by fab-10 — APPROVED** [review]

**Pull request merged**